packages/web3-app-example/package.json

```diff
@@ -0,0 +1,58 @@
+{
+	"name": "web3-app-example",
```

Comment:
I think this app shouldn't be part of our mono repo. and only sample app creation with tree shaking steps in documentation should be enough.

Reply:
If necessary you can put sample app under independent repo outside monorepo.

Reply:
here is the example web app (React)
https://github.com/ChainSafe/web3js-example-react-app